ocw-cli/cli_app.py

```diff
@@ -1232,11 +1180,11 @@ def settings_screen(header):
          if option == '4':
               screen.addstr(25, x/2, "Enter Spatial Gridding Option [Model, Observation or User]:")
               new_spatial_grid_option = screen.getstr()
-              if new_spatial_grid_option.lower() == 'model':
-                   spatial_grid_option = 'Model'
-                   spatial_grid_setting = model_lat_res
-                   note = "Spatial gridding option has changed successfully to {0}".format(spatial_grid_option)
-              elif new_spatial_grid_option.lower() == 'observation':
+              #if new_spatial_grid_option.lower() == 'model':
```

Comment:
Hey @huikyole, it seems like you've just said 2 completely different reasons for removing the code. Can you elaborate on the decision to remove this please, I don't know that I'm following. Seemingly the two reasons you mentioned are:

* It might be confusing to users if they loaded multiple models and we don't provide a means to pick which option to use.
* There are models in a coordinate system that OCW doesn't handle. Also, evidently the only models that we can possibly use with the toolkit are those that are already regridded, so there's no point to having this option. If this is the reason I'm not really understanding the first explanation you provided, so perhaps I'm not understanding you properly.